src/Http/Responses/RegisterResponse.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Stats4sd\FilamentTeamManagement\Http\Responses;
+
+use Filament\Http\Responses\Auth\Contracts\RegistrationResponse;
+
+class RegisterResponse implements RegistrationResponse
+{
+    /**
+     * @return mixed
+     */
+    public function toResponse($request)
+    {
+        // redirect user to admin panel (for admin user) or app panel (for non admin user) after user registration
+        // $home = auth()->user()->is_admin ? '/admin' : '/app';
+
+        // always redirect user to app panel, as app panel is the only entry point of this application.
+        // admin user can go to admin panel via Admin panel menu item in sidebar
+        $home = '/app';
```

Comment:
I don't think we can assume that the app will be using an 'app' panel with that exact route. Given that it looks like we do need a RegisterResponse in the package for the custom registration classes, we should instead do a 'default' here of returning the user to the current panel: `Filament::getCurrentPanel()->getHomeUrl();`

Reply:
I tried below and both return empty string...
In case there is no other way to get the URL of app panel, can we consider to put it as a config item in .env file?

```
Filament::getCurrentPanel()->getHomeUrl();

Filament::getDefaultPanel()->getHomeUrl();
```